dags/main_summary.py

```diff
@@ -23,7 +23,7 @@
     task_id="main_summary",
     job_name="Main Summary View",
     execution_timeout=timedelta(hours=10),
```

Comment:
Should we decrease the `execution_timeout` in tandem?

Reply:
There's some slight possibility that this won't actually speed the job up much - I'd rather tighten up the timeout after we see what execution looks like in the wild.